README.md

```diff
@@ -0,0 +1,28 @@
+# Regenerator
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-13-orange.svg?style=flat-square)](#contributors)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+Website to rengerate your body and soul.
+
+![demo](./IMG/demo.png)
+
+## Contributing
+
+Thank you for your interest! All types of contributions welcome. **HACK AWAY!** 🔨🔨🔨
+
+- Fork and clone this repository
+- Create your branch from the `master` branch
+- Please open your PR with the `master` branch as the base
+
+## Contributors
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```

Comment:
Add an extra line at the end

Reply:
Just added a line there. VSCode shows the line (see screenshot) but github is not showing it 
<img width="755" alt="Screen Shot 2022-10-09 at 1 40 14 PM" src="https://user-images.githubusercontent.com/39889198/194771404-064ca1c8-2820-41e8-889b-81fad6ed533b.png">